src/main/java/com/google/api/codegen/go/GoDiscoveryContext.java

```diff
@@ -41,6 +41,21 @@ public GoDiscoveryContext(Service service, ApiaryConfig apiaryConfig) {
           .put(Field.Kind.TYPE_DOUBLE, "0.0")
           .build();
 
+  @Override
+  protected String arrayTypeName(String elementName) {
```

Comment:
Where is this used? Go's array of ints is `[]int` not `intArray`. I also don't see this reflected in baseline test for any language. (I ran git diff and regexp `^\+.*Array`).